firstvoices/backend/tests/test_apis/test_sites_api.py

```diff
@@ -955,18 +955,18 @@ def test_hidden_sites_visible_in_list_only_to_team(self, team_role):
         assert len(response_data["results"]) == 0
 
     @pytest.mark.django_db
-    def test_hidden_site_detail_403(self):
+    def test_hidden_site_detail_401(self):
```

Comment:
This should apply to all APIs that handle permissions, so please fix the test here as well as in the BaseSiteContentApiTest